server/controllers/sessionController.js

```diff
@@ -51,7 +51,7 @@ sessionController.terminateSession = (req, res, next) => {
   }
 
   // destroy session and move on to next middleware
-  req.session.destroy();
+  req.session.destroy(e => console.log('error: ', e.message));
```

Comment:
Michael's change (that just got merged) will conflict with your change here, just a heads up